docs/source/GetStartedGuide.rst

```diff
@@ -140,9 +140,6 @@ The output might look like this::
 These Python files are considered by Avocado to contain ``INSTRUMENTED``
 tests.
 
-.. These should refer to proper simple tests example but they are currently
-   lacking in our tree. See GitHub issue #628.
-
```

Comment:
I'm not sure what you mean... The instrumented tests point to the packaged version of the example tests, as does this section.  On a system with avocado-examples installed:

```
# rpm -q avocado-examples
avocado-examples-0.34.0-0.fc22.noarch
# avocado list /usr/share/avocado/simpletests/
SIMPLE /usr/share/avocado/simpletests/failtest.sh
SIMPLE /usr/share/avocado/simpletests/passtest.sh
```